src/test/resources/features/FurtherDetailsTask.feature

```diff
@@ -136,3 +136,126 @@ Feature: Further Details Task page validation
     And I should see row 4 value 5 June 2026
     And I should see row 4 value 5 June 2026
+
+  Scenario: 3 - Accounting period page Error Validations
+    Given I clear the cache
+    Given Organisation User logs in as upe with credId FDTask for Pillar2
+    Then I should be on UPE business page
+    When I select option Yes and continue to next
+    Then I should be on UPE entity type page
+    When I select option UK limited company and continue to GRS page
+    Then I should navigate to UKCompany GRS page
+    When I registered successfully with BV enabled
+    And I click on Save&Continue button
+    Then I should navigate to Task list page
+    When I click Add filing member's details link
+    Then I should navigate to NFM registration page
+    When I select No option and continue to next
+    Then I should navigate to Task list page
+    And The Task Add further group details status should be Not started
+    When I click Add further group details link
+    Then I should navigate to MNE or domestic page
+    When I select option Only in the UK in further details group status page
+    And I click on Continue button
+    Then I should navigate to Group accounting period page
+    And I click on Continue button
+    Then I should see error message Enter the start date of the group’s consolidated accounting period on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period Start Day is entered as 30
+    And I click on Continue button
+    Then I should see error message Start date must include a month and year on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    And Accounting Period Start Month is entered as 01
+    And I click on Continue button
+    Then I should see error message Start date must include a day and year on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    And Accounting Period Start Year is entered as 2024
+    And I click on Continue button
+    Then I should see error message Start date must include a day and month on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period Start Day is entered as AA
+    And I click on Continue button
+    Then I should see error message Start date must include a month and year on the Group Accounting Period Start Date Page
```

Comment:
Missing Scenarios:
Start date day and month entered, but not year.
Start date day and year entered, but not month.
Start date month and year entered, but not year.